dashboard/test/controllers/ops/cohorts_controller_test.rb

```diff
@@ -174,9 +185,16 @@ def teacher_params
       # district info is included
       expected_district = {'name' => @district.name, 'id' => @district.id, 'location' => @district.location}
       assert_equal expected_district, cohort_json['teachers'].first['district']
+
+      # no notification to the ops team
+      assert ActionMailer::Base.deliveries.collect(&:subject).none? {|subject| subject.include? '[ops notification]'}
```

Comment:
If you want to disable this warning add the following to [.rubocop.yml](https://github.com/code-dot-org/code-dot-org/blob/staging/.rubocop.yml):
```yml
Style/CollectionMethods:
  Enabled: false
```
I don't have a strong preference either way, but it might be nice to standardize on a consistent set of `Enumerable` operators.  For example I have an easier time parsing `find` than `detect`.